### PR TITLE
[ci] Add Verdi Raft with dependencies to CI

### PR DESCRIPTION
**Pull request by palmskog**
**Kind:** infrastructure.

Note that two of the dependencies (but not Verdi Raft itself) depends on positive co-inductive types. We hope to address this in the near future.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -509,6 +509,9 @@ library:ci-unimath:
 library:ci-vst:
   <<: *ci-template-flambda
 
+library:ci-verdi-raft:
```
Comment (ejgallego) [inline .gitlab-ci.yml]: Please keep the entries in alphabetical ordering.

**Review by ejgallego — APPROVED** [review]
LGTM, in the future we may want to split some deps off; but for now things are good IMHO.

File: .gitlab-ci.yml (hunk 1 above)
Comment (palmskog) [reply .gitlab-ci.yml]: Fixed.

**Comment by silene** [comment]
I won't be backporting to 8.9 a CI test that depends on a master branch.

**Pull request merged**